integrationtests.py

```diff
@@ -151,6 +151,16 @@ def testRecompileObjectSetOtherDir(self):
         subprocess.check_call(cmd) # Compile once
         subprocess.check_call(cmd) # Compile again
 
+    def testPipedOutput(self):
+        with cd(ASSETS_DIR):
+            cmd = [PYTHON_BINARY, CLCACHE_SCRIPT, "/nologo", "/c", "fibonacci.c", "/Fofibonacci_c.obj"]
```

Comment:
Actually there is an output, The test fails when I leave the old code.
But it is better to have more output. I will remove /nologo